cypress.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from "cypress";
+import plugin from "./cypress/plugins";
+
+export default defineConfig({
+  env: {
+    HOSTNAME: "http://localhost:3000",
```

Comment:
should be this hardcoded?

Reply:
I guess no. But I don't know why it has been hardcoded(It same in the old config).

Reply:
I guest that if someone wants to change the HOSTNAME and PORT, should do from an env variable rather that a typescript class, but we can manage it in another issues

Reply:
according to the doc, we should rather use[ the  baseUrl param](https://docs.cypress.io/guides/references/best-practices#Setting-a-Global-baseUrl)

Reply:
@gasp could you open an issue for this and propose a PR linked to that issue? thanks!